pkg/operator/operator.go

```diff
@@ -37,6 +37,7 @@ import (
 	"github.com/aws/aws-sdk-go/service/eks/eksiface"
 	"github.com/aws/aws-sdk-go/service/iam"
 	"github.com/aws/aws-sdk-go/service/ssm"
+	prometheusv1 "github.com/jonathan-innis/aws-sdk-go-prometheus/v1"
```

Comment:
Is this meant to be from your fork?

Reply:
To inject the prometheus client metrics

Reply:
See: https://github.com/jonathan-innis/aws-sdk-go-prometheus/blob/main/v1/v1.go